docs/new-audits.md

```diff
@@ -31,8 +31,28 @@ An audit can return a number of different [detail types](https://github.com/Goog
 | `'url'`                   | Network Resource      | we will make it a pretty link          |
 | `'thumbnail'`             | Image Resource        | same as above, but we show a thumbnail |
 | `'link'`                  | -                     | arbitrary link / url combination       |
+| `'bytes'`                 | -                     | value is in bytes but formatted as KiB |
 | `'text'\|'ms'\|'numeric'` | -                     |                                        |
 
+### Granularity
+
+The following detail types accept a `granularity` field:
+
+- `bytes`
+- `ms`
+- `numeric`
+
+`granularity` must either be between 0 - 1 (excluding zero), or any positive integer power of 10. Some examples of valid values for `granularity`:
+
+- 0.001
+- 0.01
+- 0.1
+- 0.5
+- 1
+- 10
+- 100
```

Comment:
thoughts on removing the restriction about "power of 10"? might make a transition away from granularity more difficult at a later date..

(allowing any positive number)

Reply:
Personally I'd rather restrict this to any +/- integer power of 10. The 0.5 case feels out of place.

That being said, if it aint broke I don't really have a strong opinion.

Reply:
> Personally I'd rather restrict this to any +/- integer power of 10. The 0.5 case feels out of place.

+1 We've only incidentally handled the 0.5 case and have never actually used it.

Reply:
Done

Reply:
Rm from this doc list too

Reply:
oops